engine/runtime/src/main/java/org/enso/compiler/TreeToIr.java

```diff
@@ -356,28 +359,34 @@ private List<IR.DefinitionArgument> translateArgumentsDefinition(java.util.List<
     * @param body the body to be translated
     * @return the [[IR]] representation of `body`
     */
-  private scala.collection.immutable.List<IR> translateTypeBody(java.util.List<Line> block, boolean found) {
-    var ir = block.stream().map(line -> translateTypeBodyExpression(line)).filter(line -> line != null).iterator();
-    return CollectionConverters.asScala(ir).toList();
+  private List<IR> translateTypeBody(java.util.List<Line> block, boolean found) {
```

Comment:
What's `found` for?

Reply:
Debris from the past. Removed in [d6fc9e2](https://github.com/enso-org/enso/pull/3806/commits/d6fc9e2c8a3d2afd3b4d66357a9794baa1ff945a).